services/models/destination_policy.go

```diff
@@ -4,10 +4,32 @@ import (
 	"github.com/kiali/kiali/kubernetes"
 )
 
+// DestinationPolicies destinationPolicies
+//
+// This is used for returning an array of DestinationPolicies
```

Comment:
Minor suggestion about style.
For documentation I find useful to be explicit and about pronoums.
Instead "This is used" ... > "This type/endpoint/function/... is used for" 
That would help to avoid ambiguity.
It is more verbose but at the end it helps.
(Same for other areas)